File: algorithmic_toolbox/week3_greedy_algorithms/1_money_change/change.go
package main

import (
	"C"
	"fmt"
)

func getChange(m int) int {
	n := m % 10
	o := n % 5

	return m/10 + n/5 + o/1
}

func main() {
	var m int
	fmt.Scan(&m)
	fmt.Println(getChange(m))
}

File: algorithmic_toolbox/week3_greedy_algorithms/1_money_change/change.py
import sys


def get_change(m: int) -> int:
    n = m % 10
    o = n % 5

    return m // 10 + n // 5 + o // 1


if __name__ == "__main__":
    m = int(sys.stdin.read())
    print(get_change(m))
